src/sonic-bgpcfgd/bgpcfgd/managers_static_rt.py

```diff
@@ -60,19 +71,47 @@ def set_handler(self, key, data):
 
         if cmd_list:
             self.cfg_mgr.push_list(cmd_list)
-            log_debug("Static route {} is scheduled for updates".format(key))
+            log_debug("{} Static route {} is scheduled for updates. {}".format(self.db_name, key, str(cmd_list)))
         else:
-            log_debug("Nothing to update for static route {}".format(key))
+            log_debug("{} Nothing to update for static route {}".format(self.db_name, key))
 
         self.static_routes.setdefault(vrf, {})[ip_prefix] = (ip_nh_set, route_tag)
 
         return True
 
 
+    def skip_appl_del(self, vrf, ip_prefix):
```

Comment:
May be it is good to provide some comments as to what this function does and why it is skipped for some cases.

Reply:
@baorliu , would you please address this for completion?

Reply:
> 

will update it soon...